src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php

```diff
@@ -68,10 +68,12 @@ public function process(ContainerBuilder $container)
                     continue;
                 }
 
-                if (false !== strpos($definedTag, $tag) || levenshtein($tag, $definedTag) <= strlen($tag) / 3) {
-                    $candidates[] = $definedTag;
+                if (false !== strpos($definedTag, $tag) || $lev = levenshtein($tag, $definedTag) <= strlen($tag) / 3) {
+                    $candidates[$definedTag] = $lev;
```

Comment:
why not just `$candidates[$definedTag] = true;`? That would avoid creating the `$lev` variable that is not used anyway.